### PR TITLE
Add facets to ESI funds

### PR DESCRIPTION
**Pull request by jackscotti**
fund_state, fund_type, location, funding_source, closing_date

ticket:https://www.pivotaltracker.com/story/show/87188568

### CODE REVIEW

File: app/presenters/finder_content_item_presenter.rb [hunk 1]
```diff
@@ -48,6 +48,7 @@ def details
       format_name: metadata.fetch("format_name", nil),
       signup_link: metadata.fetch("signup_link", nil),
       show_summaries: metadata.fetch("show_summaries", false),
+      summary: metadata.fetch("summary", nil),
```
Comment (tommyp) [inline app/presenters/finder_content_item_presenter.rb]: Can you do this in a separate commit? Maybe even in a separate PR? It's not related to the ESI funds specifically.

Comment (jackscotti) [inline app/presenters/finder_content_item_presenter.rb]: :+1:
